core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1798,6 +1913,12 @@ public function extractVimeoId($varValue, Contao\DataContainer $dc)
 	 */
 	public function toggleIcon($row, $href, $label, $title, $icon, $attributes)
 	{
+		// Disable the button if the element type is not allowed
```

Comment:
does this permission need to be checked in the ajax callback as well?

Reply:
Yes. Added in 2f36280fb87d24e29a945336b2d49e739a884a1b.